src/dmd/backend/mscoffobj.c

```diff
@@ -1043,12 +1043,17 @@ void MsCoffObj::startaddress(Symbol *s)
 
 bool MsCoffObj::includelib(const char *name)
 {
-    int seg = seg_drectve();
-    //dbg_printf("MsCoffObj::includelib(name *%s)\n",name);
-    SegData[seg]->SDbuf->write(" /DEFAULTLIB:\"", 14);
-    SegData[seg]->SDbuf->write(name, strlen(name));
-    SegData[seg]->SDbuf->writeByte('"');
-    return true;
+    if (name && strlen(name) > 0)
```

Comment:
This isn't checked in cgobj.c, why here? What does it have to do with Throwable?

Reply:
It's because the tests are compiled with `defaultlib=`, and the mircosoft linker rejects empty arguments to `/DEFAULTLIB:`.  It's just necessary to get the tests to pass on Windows.

Reply:
BTW, in glue.d where this function is called:
```
if (global.params.mscrtlib && global.params.mscrtlib[0])
      objmod.includelib(global.params.mscrtlib);
```
so now the check is being done twice. Also, the `strlen` can be safely replaced with `[0]`.